src/view/SignupView.form

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should consider creating a general form layout that we can pass fields, titles, etc. to, but this is fine for now.

Reply:
Yeah, one of the tricky parts about intellij forms is definitely components... I tried to make chatview a component and failed pretty badly (nothing would render).

Reply:
hmm ok